### ISSUE TITLE
Disable simultaneous preview playback

### ISSUE THREAD

**Issue by AxonFlux** [MTG/freesound#1455]
Currently, when previewing a sound in a results list, it does not stop playing when clicking on the preview button for a new sound. This forces users to click pause in order to hear another sound which is counterintuitive and quite sloppy UI behaviour considering it adds another unnecessary click. 

If this is possible with SoundManager2, previous sound should automatically pause playback when previewing the next sound. This would streamline the process of previewing many sounds in succession.

**Comment by bdejong** [comment]
Many of our users actually see this as a feature. They like playing sounds together to hear how they blend... The best way to deal with this would be to add some user preferences and let them default to the current behavior.

**Comment by ffont** [comment]
Yeah, I was going to answer the same. We should have a preferences checkbox because this is not the first time this suggestion is made.

**Comment by ffont** [comment]
I suggest to do this in the new frontend, as the player code is completely different. FYI: We're trying now to re-activate the new front end development because the guy working on the new frontend has not been able to do much lately and we're in the process of finding someone new.

**Comment by AxonFlux** [comment]
Good call. I was considering this as an option instead. That would be ideal.

**Comment by Combinacijus** [comment]
Any news on this? I'd like this feature

**Comment by ffont** [comment]
I'm actively working on the new frontend and will look into further player updates soon. I guess it won't be done anytime soon, but hopefully sometime this year.

**Comment by ffont** [comment]
Closing this issue as this feature was added in the new UI some months ago.
